clusterloader2/testing/batch/config.yaml

```diff
@@ -17,6 +17,8 @@
 {{$largeJobSize := 400}}
 {{$largeJobsPerNamespace := DivideInt $podsPerNamespace (MultiplyInt 4 $largeJobSize)}}
 
+{{$jobRunningTime := "30s"}}
```

Comment:
worth having a `DefaultParam` here, so we can override it without editing this file.

Reply:
Done